dpnp/backend_statistics.pyx

```diff
@@ -166,7 +189,26 @@ cpdef dparray dpnp_max(dparray input, axis):
     return dpnp_result_array
 
 
+cpdef dparray _dpnp_mean(dparray input):
+    cdef DPNPFuncType param1_type = dpnp_dtype_to_DPNPFuncType(input.dtype)
+    cdef DPNPFuncType output_type = dpnp_dtype_to_DPNPFuncType(numpy.float64 if input.dtype != numpy.float32 else numpy.float32)
```

Comment:
Isn't it resolved by `fmap[DPNPFuncName::DPNP_FN_MEAN][eft_FLT][eft_FLT] = {eft_FLT, (void*)custom_mean_c<float, float>};`?